### ISSUE TITLE
[Content organization] Your page doesn't follow heading hierarchy. Headings aren't properly nested, and headings and/or labels aren't relevant to the page. (02.04.3)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#30024]
#### VFS team name
eBenefits

#### VFS product name
Dependency Verification

#### Point of Contact/Reviewers
Josh Kim (Accessibility)

---

### Platform Issue
Your page doesn't follow heading hierarchy. Headings aren't properly nested, and headings and/or labels aren't relevant to the page.

### Issue Details
Add or remove a dependent on your VA disability benefits confirmation page has no heading structure.

### Link, screenshot or steps to recreate

<img width="1260" alt="confirmation-page-headings" src="https://user-images.githubusercontent.com/14154792/133645727-720045c0-a251-49fd-aca8-9bf4ce396d08.png">

### VA.gov Experience Standard

[Category Number 02, Issue Number 04](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 1.3.1_A
WCAG SC 2.4.6_AA

---

### Platform Recommendation
All bolded `p` tags should be reassigned to proper heading levels. See screenshot for more.

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] VFS team closes the ticket when the issue has been resolved
